Assets/Scripts/Features/Gameplay/Take Photo/GeneratePhoto.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// A script to generate a photograph.
+/// </summary>
+public class GeneratePhoto : MonoBehaviour
+{
+    [SerializeField] private GameObject _picture;
+    [SerializeField] private RenderTexture _renderTexture;
+
+    void Start() {
+        //PhotoCameraApi.OnCreatePhoto += GenerateNewPhoto;
+    }
+
+    /// <summary>
+    /// Create a new photograph.
+    /// </summary>
+    void GenerateNewPhoto() {
+       Renderer rend = _picture.GetComponent<Renderer>();
+        rend.material.mainTexture = ToTexture2D(_renderTexture);
+    }
+
+    /// <summary>
+    /// Returns a Texture2D generated from RenderTexture "rTex".
+    /// </summary>
+    /// <param name="rTex"></param>
+    /// <returns></returns>
```

Comment:
Add small description of "rTex" parameter
Also add what it returns in the comment, not just <returns></returns>